src/workloads/query/TimeseriesStressUnpacking.yml

```diff
@@ -0,0 +1,179 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This test stress tests bucket unpacking. Each document is large with many top-level fields.
```

Comment:
Thinking about the SBE's specifics of unpacking, I think we need to cover one more parameter here, namely, how many fields need to be unpacked from each bucket. Something like: 1, 10, 20. It might also be of interest to compare queries that project _all_ fields vs kExclude that project none but have to unpack everything anyway (these queries will run in classic).

Reply:
Done!